File: app/src/main/java/com/example/dpstuffprovider/models/OrderComposData.kt
package com.example.dpstuffprovider.models

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class OrderComposData (
    @SerializedName("name") var name : String,
    @SerializedName("shopName") var shopName : String,
    @SerializedName("quantity") var quantity : Int,
    @SerializedName("price") var price : Int
) : Serializable

data class OrderCompos(
    var listOrderCompos : List<OrderComposData>
) : Serializable

File: DPSP Api v2/Models/Data/OrderComposData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace DPSPApiV2.Models.Data
{
    public class OrderComposData
    {
        public int ProductId { get; set; }
        public string Name { get; set; }
        public float Price { get; set; }
        public string Vendor { get; set; }
        public int Quantity { get; set; }
    }
}
